specs/takanon/extraction/חוק-יסוד_הכנסת_סעיף_10.md

```diff
@@ -0,0 +1,11 @@
+# חוק-יסוד_הכנסת
+
+## חוק־יסוד: הכנסת
+
+### סעיף 10
+
+[10.](https://he.wikisource.org/wiki/%D7%97%D7%95%D7%A7-%D7%99%D7%A1%D7%95%D7%93:_%D7%94%D7%9B%D7%A0%D7%A1%D7%AA#%D7%A1%D7%A2%D7%99%D7%A3_10)
```

Comment:
Not to repeat for every file here, but this should be:

`https://he.wikisource.org/wiki/חוק-יסוד:_הכנסת#סעיף_10`

Also, the link reference text is not right - "10." is not a good text - it should be something like "סעיף 10: יום הבחירות - שבתון".

Reply:
Also, the title of the item - ״יום הבחירות - שבתון״ should be highlighted somehow, or incorporated in the structure of the item (e.g. instead of `### סעיף 10`, use `### סעיף 10 - יום הבחירות שבתון`